utils/google.js

```diff
@@ -406,127 +426,4 @@ module.exports = {
       });
     })
   },
-
```

Comment:
it's kinda sad to see this all deleted :cry:

Reply:
it's better this way. Rest in peace locking code

Reply:
💔